presto-product-tests/bin/run_on_docker.sh

```diff
@@ -114,42 +114,46 @@ function terminate() {
   exit 130
 }
 
+function getAvailableEnvironments() {
+AVAILABLE_ENVIRONMENTS=`ls $ENVIRONMENT_LOCATION -I files`
```

Comment:
Or instead of using function you can use just variable. But my point is that there is no need to use both of the approaches.